eve/main.yml

```diff
@@ -2467,6 +2467,7 @@ stages:
     worker:
       type: local
     steps:
+      - ShellCommand: *add_final_status_artifact_failed
```

Comment:
Is this related to this PR (I see it appears in the `generate-snapshot` stage)? Wasn't this part of #3084 already?

Reply:
Hmm weird, I'm not sure I did put this in this stage... Maybe due to a rebase ? (or I just made a mistake).
Anyway, it's definitely not related to this PR and as you commented, it has already been fixed, I'll remove this line.

Reply:
done